.github/workflows/test-package.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.9", "3.10", "3.11", "3.12"]
+        python-version: ["3.11", "3.12"]
```

Comment:
Ah yeah good point. Let's bump the nwa-stdlib version to 1.6.0 for this?

Reply:
1.6.0 it is